src/sidebar/store/modules/annotations.js

```diff
@@ -137,14 +152,58 @@ const update = {
     };
   },
 
+  CLEAR_ANNOTATIONS: function () {
+    return { annotations: [] };
```

Comment:
Probably a good idea to reset `focused` and `highlighted` here to prevent an inconsistent state where an annotation ID appears in `focused` or `highlighted` but no longer exists.

Reply:
Good catch.